### PR TITLE
[ADD] discuss: doc about email template

### PR DESCRIPTION
**Pull request by kbr-odoo**
FYI: The whole documentation has been re-written and it is being reviewed. That means the whole structure will be updated.

### CODE REVIEW

File: discuss/advanced/emailtemplate.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+   prior knowledge about placeholders.**
+
+To add a new one, click on *Create* and choose the type of document this template is used with. In
+the example below, the template would be sent to job applicants.
+
+.. image:: media/newtemplate.png
+   :align: center
+   :alt: New email template form in Odoo
+
+| Under *Email Configuration*, fields such as *From*, *To (Emails)*, *To  (Partners)*, require
```
Comment (Lau-Odoo) [inline discuss/advanced/emailtemplate.rst]:
```suggestion
| Under *Email Configuration*, fields such as *From*, *To (Emails)*, *To (Partners)*, require
```

File: discuss/advanced/emailtemplate.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  placeholders. If the *From* field is not set, the default value is the author’s email alias if
```
Comment (Lau-Odoo) [inline discuss/advanced/emailtemplate.rst]:
```suggestion
  placeholders. If the *From* field is not set, the default value is the author’s email alias, if
```

File: discuss/advanced/emailtemplate.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+-------------------------------------------------
+
+Under the tab *Dynamic Placeholder Generator*, look for the *Field* you would like to use.
+
+.. image:: media/placeholders.png
+   :align: center
+   :alt: View of the dynamic placeholder generator tab under a new template in Odoo
+
+Next, copy the *Placeholder Expression* and past it on the *Body* of the email, under the *Content*
+tab, using (essentially) the *Code View*.
```
Comment (Lau-Odoo) [inline discuss/advanced/emailtemplate.rst]:
```suggestion
Next, copy the *Placeholder Expression* and paste it in the *Body* of the email, under the *Content*
tab, using (essentially) the *Code View*.
```

**Comment by kbr-odoo** [comment]
@Lau-Odoo thank you!

File: discuss/advanced/emailtemplate.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
+===============
+Email Templates
+===============
+
+A nice written email can mean the difference between total disregard of the message or an excited
+reply. And not just that, it lets you save time, stay consistent, and deliver personalized messages,
+allowing customers and prospects to have the best brand experience.
```
Comment (jubodoo) [inline discuss/advanced/emailtemplate.rst]: This doesn't make sense. 

A nice written email doesn't make you save time. The template does, but there's no talk about templates before so it's confusing. 

I would keep some of it but rework it to include that it's about templates.

File: discuss/advanced/emailtemplate.rst [hunk 5]
```diff
@@ -0,0 +1,57 @@
+===============
+Email Templates
+===============
+
+Using email templates let you save time and stay consisted, as you eliminate the need of having to
+rewrite entire emails from scratch for every send. Templates also allow the delivery of
+personalized messages, offering to your recipients a better brand experience.
```
Comment (jubodoo) [inline discuss/advanced/emailtemplate.rst]: I still don't think it's good. 

And there's English mistakes as well 

Here's an idea:

We all know writing good emails is vital to get a high response, but you don't want to rewrite the same structure every time you need to send an email, do you? That's where email templates come in, without the need to rewrite the entire email structure every time you save time to focus on the content. 

Multiple templates will also let you deliver the right message to the right audience, improving their overall experience with the company.

**Comment by kbr-odoo** [comment]
@jubodoo
Thank you for your help.

**Pull request merged**